core/lib/server/render/pub/renderMarkdownWithPub.ts

```diff
@@ -37,12 +38,19 @@ const visitValueDirective = (node: NodeMdast & Directive, context: utils.RenderW
 	const field = expect(attrs.field, "Missing field attribute in value directive");
 
 	let value: unknown;
+	let pub: utils.RenderWithPubPub;
 
 	if (attrs?.rel === "parent") {
 		const parentPub = expect(context.parentPub, "Missing parent pub");
-		value = parentPub.values[field];
+		pub = parentPub;
 	} else {
-		value = context.pub.values[field];
+		pub = context.pub;
+	}
+
+	if (field === "title") {
+		value = getPubTitle(pub);
```

Comment:
nice, this way #822 will just take care of correctly adding the title.
depending on the merge order, we will need to add `pubType` information on this pub however.